config/config-dev.json

```diff
@@ -235,6 +235,106 @@
+                },
+                "Type": "edx"
+            },
+            {
+                "Name": "BrightScout",
+                "OAuthConsumerKey": "brightscout-test_1044",
+                "OAuthConsumerSecret": "secret",
+                "PersonalChannels": {
+                    "ChannelList": {
+                        "capstone": {
+                            "IdProperty": "custom_team_id",
+                            "NameProperty": "custom_team_name"
+                        },
+                        "plg": {
+                            "IdProperty": "custom_cohort_id",
+                            "NameProperty": "custom_cohort_name"
+                        }
+                    },
+                    "Type": "custom-properties"
+                },
+                "Teams": {
+                    "course-v1:appsembler-the-red+Riff+101": "riff",
+                    "test-context-1": "riff-a",
+                    "test-context-2": "riff-b"
+                },
+                "Type": "edx"
+            },
+            {
+                "Name": "Emeritus",
+                "OAuthConsumerKey": "canvas-emeritus_5863",
+                "OAuthConsumerSecret": "emeritus secret",
+                "PersonalChannels": {
+                    "AuthorizationToken": "dummy~value",
+                    "ChannelList": {
+                        "capstone": {
+                            "NameRegEx": "^riff_Team_"
+                        },
+                        "plg": {
+                            "NameRegEx": "^riff_PLG_"
+                        }
+                    },
+                    "Type": "emeritus-groups"
+                },
+                "Teams": {
+                    "eabb2ed57cf5dec85996803535cbccb4c7f62492": "sloan-1"
+                },
+                "Type": "canvas"
+            },
+            {
+                "Name": "ATestName",
+                "OAuthConsumerKey": "brightscout-test_1044",
+                "OAuthConsumerSecret": "secret",
+                "PersonalChannels": {
+                    "ChannelList": {
+                        "breakfastsquad": {
+                            "IdProperty": "custom_team_id",
+                            "NameProperty": "custom_team_name"
+                        },
+                        "capstone": {
+                            "IdProperty": "custom_team_id",
+                            "NameProperty": "custom_team_name"
+                        },
+                        "plg": {
+                            "IdProperty": "custom_cohort_id",
+                            "NameProperty": "custom_cohort_name"
+                        }
+                    },
+                    "Type": "custom-properties"
+                },
+                "Teams": {
+                    "test-context-1": "brecs-team",
+                    "test-context-2": "dougs-team"
+                },
+                "Type": "edx"
+            }
+        ]
+    },
```

Comment:
I think this LTISettings section should not be committed. It is good for your testing, and you probably should keep it in your local config.json, but the setting are very fake and somewhat invalid and not something that I think should be distributed.

Reply:
Sorry, didn't mean to commit that.